policy/release/schedule.rego

```diff
@@ -64,10 +66,19 @@ deny contains result if {
 #   - policy_data
 #
 deny contains result if {
+	# (For this one let's do it always)
 	some error in _rule_data_errors
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# We want these checks to apply only if we're doing a release. Detect that by checking
+# the `pipeline_intention` value which is set to "release" for Konflux release pipelines.
+default _schedule_restrictions_apply := false
+
+_schedule_restrictions_apply if {
+	lib.rule_data("pipeline_intention") == "release"
```

Comment:
It would be great to include something in the docs that indicate that these rules only apply if `pipeline_intention` is set to `release`.

Reply:
Thanks, good idea.

Reply:
Done in latest revision.